tools/test/test.bzl

```diff
@@ -1,11 +1,13 @@
 load("@io_bazel_rules_kotlin//kotlin:jvm.bzl", "kt_jvm_test")
 load(":runtime_resources.bzl", "runtime_resources")
 
+DEFAULT_SRC_SETS = ["src/test/java", "src/test/kotlin"]
```

Comment:
This could be made private by `_DEFAULT_SRC_SETS` for now.

Reply:
changed it in new commit. Thanks!